apps/api/src/app/controllers/__tests__/createTestModule.ts

```diff
@@ -43,5 +44,10 @@ export default async (testDatabaseName: string) =>
             },
         ],
 
-        controllers: [ResourceViewModelController, EdgeConnectionController, TagController],
+        controllers: [
+            ResourceViewModelController,
+            EdgeConnectionController,
+            TagController,
+            CategoryController,
```

Comment:
Can we pass these in instead? It seems like we keep updating this every time we add a controller.

Maybe there's a way to use the actual App Module?